R/pdf_array_job.R

```diff
@@ -42,15 +42,16 @@ pdf_array_job <- function(jobs_file, r_script, final_out_dir,
   
   jobs = data.table::fread(jobs_file)
   command <- paste0(
-    "qsub", 
-    " -l fthread=", fthread, ",fmem=", fmem, ",h_rt=", h_rt, 
-    " -t 1:", nrow(jobs), 
-    " -q ", queue, 
-    " -P ", project, 
-    " -N ", job_name, 
-    " -now no",
-    " -o ", output, 
-    " -e ", errors,
+    "sbatch", 
+    " --mem ", fmem,
+    " -c ", fthread, 
+    " -t ", h_rt, 
+    " --array=1-", nrow(jobs), 
```

Comment:
This allows for all array jobs to be launched simultaneously (assuming no other cluster constraints)

I'm not sure if this is a problem or not.

If you want to add a max limit you do so with this arg. "--array=0-15%4" will create an array of 16 jobs limited to 4 running simultaneously

Reply:
It's a good question... it's never been a concern before. I think generally people use this to launch one job per location in the hierarchy, which is a manageable amount of jobs.